src/routes/blog/[[page]]/+page.svelte

```diff
@@ -34,6 +41,51 @@
         });
     });
 
+    let selectedCategory = $page.url.searchParams.get('category') ?? 'Latest';
+
+    const handleSearch = async (value: string) => {
```

Comment:
The argument here is always the value from the global query right? Why not make this function without an argument, and use the global `query`? Line 47 would then become something like `const queryLowercase = query.toLowerCase()`

Reply:
ohh yeah that makes sense.